examples/async-sign/signed_message/src/lib.rs

```diff
@@ -37,7 +37,11 @@ pub unsafe extern "C" fn handle() {
             msg::reply_bytes(b"Uninitialized", 10_000, 0);
             return;
         }
-        Some(s) => s.0,
+        Some(s) => {
+            let mut arr = [0u8; 32];
+            arr.copy_from_slice(s.as_ref());
+            arr
+        }
```

Comment:
Why we've made the code more difficult? It is ok for the `newtype` idiom to get the inner value:

https://doc.rust-lang.org/stable/rust-by-example/generics/new_types.html

Reply:
Depend on our decision about `ActorId` constructor